docs/source/user-guide/operators.md

```diff
@@ -19,15 +19,18 @@
 
 # Supported Spark Operators
 
-The following Spark operators are currently available:
+The following Spark operators are currently replaced with native versions. Query stages that contain any operators
+not supported by Comet will fall back to regular Spark execution.
 
-- FileSourceScanExec/BatchScanExec for Parquet
-- Projection
-- Filter
-- Sort
-- Hash Aggregate
-- Limit
-- Sort-merge Join
-- Hash Join
-- Shuffle
-- Expand
+| Operator                                     | Notes                                  |
+| -------------------------------------------- | -------------------------------------- |
+| FileSourceScanExec/BatchScanExec for Parquet |                                        |
+| Projection                                   |                                        |
+| Filter                                       |                                        |
+| Sort                                         |                                        |
+| Hash Aggregate                               |                                        |
+| Limit                                        |                                        |
+| Sort-merge Join                              | Sort-merge join is disabled by default |
```

Comment:
oh, is SMJ disabled by default?

Reply:
It looks like I misunderstood this. I have update this.